service/controller/aws/v22/version_bundle.go

```diff
@@ -27,6 +27,21 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Migrated to use default app catalog.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "kiam",
+				Description: "Add managed kiam app into default app catalog(aws only).",
+				Kind:        versionbundle.KindAdded,
+			},
+			{
+				Component:   "external-dns",
+				Description: "Add managed external-dns app into default app catalog.",
+				Kind:        versionbundle.KindAdded,
+			},
+			{
+				Component:   "cert-manager",
+				Description: "Add managed cert-manager app into default app catalog.",
+				Kind:        versionbundle.KindAdded,
+			},
```

Comment:
These should be added to the clusterapi version bundle as this is for legacy. Sorry I missed it in my earlier review :(

https://github.com/giantswarm/cluster-operator/blob/bab03725d7ecc7413b79036a14939dcd10d4ed9a/service/controller/clusterapi/v22/version_bundle.go

Reply:
Good catch, updated